google/cloud/forseti/services/scanner/dao.py

```diff
@@ -54,15 +55,16 @@ class Violation(base):
         __tablename__ = violations_tablename
 
         id = Column(Integer, primary_key=True)
+        full_name = Column(String(1024))
```

Comment:
Just a note: will you add a `timestamp` or a `violation_timestamp` column in this PR?

Reply:
Let me follow-up with a second PR for this.